VimeoNetworking/Sources/VimeoRequestSerializer.swift

```diff
@@ -169,4 +180,33 @@ final public class VimeoRequestSerializer: AFJSONRequestSerializer
         
         return request
     }
+    
+    private func requestModifyingUserAgentHeader(fromRequest request: URLRequest) -> URLRequest
+    {
+        guard let frameworkVersion = Bundle(for: type(of: self)).infoDictionary?["CFBundleShortVersionString"] as? String else
```

Comment:
Curious if we have to use `type(of: self)` here or if it'll allow us to just do `Bundle(for: VimeoRequestSerializer)`. Would that work?